README.md

```diff
@@ -190,6 +206,19 @@ system:
   name: vmhost
 ```
 
+## Development
+
+After checking out the repo, run `bin/setup` to install
+dependencies. Then, run `rake spec` to run the tests. You can also run
+`bin/console` for an interactive prompt that will allow you to
+experiment.
+
+To install this gem onto your local machine, run `bundle exec rake
+install`. To release a new version, update the version number in
+`version.rb`, and then run `bundle exec rake release`, which will
+create a git tag for the version, push git commits and tags, and push
+the `.gem` file to [rubygems.org](https://rubygems.org).
```

Comment:
Is there a point to pasting generic release instructions which will not work with our processes into this specific file?